toolkit/scalebar/src/main/java/com/arcgismaps/toolkit/scalebar/Scalebar.kt

```diff
@@ -141,6 +143,7 @@ private fun ShowScalebar(
             maxWidth = maxWidth.toFloat(),
             label = labels[0].label,
             colorScheme = colorScheme,
+            labelTypography = labelTypography
```

Comment:
we should only pass what we need, which is the `TextStyle`
```suggestion
            labelStyle = labelTypography.labelStyle
```

Reply:
How does it even matter here, `labelTypography` just has one property. If we do it here, we will have to do it for `ScalebarColors` and `ScalebarShapes` too, which will unnecessarily increase the parameters being passed.

Reply:
good point 👍🏼